jsonnet/rules.jsonnet

```diff
@@ -75,15 +75,15 @@
             record: 'cluster:capacity_memory_bytes:sum',
           },
           {
-            expr: 'sum(1 - rate(node_cpu_seconds_total{mode="idle"}[1m]) * on(namespace, pod) group_left(node) node_namespace_pod:kube_pod_info:{})',
+            expr: 'sum(1 - rate(node_cpu_seconds_total{mode="idle"}[2m]) * on(namespace, pod) group_left(node) node_namespace_pod:kube_pod_info:{})',
```

Comment:
let's go with `[5m]` as @brancz suggested out of band.

Reply:
This is not a cadvisor metric, but node_exporter one, so no need for 5m.